code/cfile/cfile.cpp

```diff
@@ -1015,20 +1000,16 @@ static CFILE *cf_open_packed_cfblock(const char* source, int line, FILE *fp, int
 		return NULL;
 	} else {
 		CFILE *cfp;
-		Cfile_block *cfbp;
-		cfbp = &Cfile_block_list[cfile_block_index];
-	
-		cfp = &Cfile_list[cfile_block_index];
-		cfp->id = cfile_block_index;
-		cfp->version = 0;
-		cfbp->data = NULL;
-		cfbp->fp = fp;
-		cfbp->mem_mapped = false;
-		cfbp->dir_type = type;
-		cfbp->max_read_len = 0;
-
-		cfbp->source_file = source;
-		cfbp->line_num = line;
+		cfp = &Cfile_block_list[cfile_block_index];
```

Comment:
Wouldn't it be cleaner to just declare and define it on one line?

Reply:
Indeed it would. I made a lot of bulk changes and must have missed this.